spec/models/claims/claim_spec.rb

```diff
@@ -110,4 +110,41 @@
       end
     end
   end
+
+  describe "#ready_to_be_checked?" do
+    it "returns true if the claim is ready to be checked" do
+      provider = create(:provider)
+      mentor = create(:claims_mentor)
+      claim = create(:claim, provider:)
+      create(:mentor_training, mentor:, provider:, claim:, hours_completed: 20)
+
+      expect(claim.ready_to_be_checked?).to eq(true)
+    end
+
+    it "returns false if the claim does have mentors" do
+      claim = build(:claim)
+
+      expect(claim.ready_to_be_checked?).to eq(false)
+    end
+
+    it "returns false if the claim does have mentor training hours" do
+      provider = create(:provider)
+      mentor = create(:claims_mentor)
+      claim = create(:claim, provider:)
+      create(:mentor_training, mentor:, provider:, claim:, hours_completed: nil)
+
+      expect(claim.ready_to_be_checked?).to eq(false)
+    end
+  end
+
+  describe "#remove_mentor_trainings_without_hours" do
+    it "removes all mentor trainings from a claim" do
+      claim = create(:claim)
+      create(:mentor_training, claim:)
+      create(:mentor_training, claim:)
```

Comment:
This test doesn't show that it keeps trainings _with_ hours.

If I changed

```diff
class Claims::Claim < ApplicationRecord
  def remove_mentor_trainings_without_hours
-   mentor_trainings.without_hours.destroy_all
+   mentor_trainings.destroy_all
  end
end
```

The unit test here would still pass.